python/python/glide/async_commands/core.py

```diff
@@ -1100,6 +1100,25 @@ async def ttl(self, key: str) -> int:
         """
         return cast(int, await self._execute_command(RequestType.TTL, [key]))
 
+    async def type(self, key: str) -> str:
+        """
+        Returns the string representation of the type of the value stored at `key`.
+
+        See https://redis.io/commands/type/ for more details.
+
+        Args:
+            key (str): The key to check its data type.
+
+        Returns:
+            Optional[str]: The type of value stored at `key`, or "none" when key doesn't exist.
+            The different types that can be returned are: string, list, set, zset, hash and stream.
```

Comment:
Lets remove this line as it would require us to maintain it with new types will be added to redis, and, also - when modules are being loaded we might have more types